dev/com.ibm.ws.javaee.dd/src/com/ibm/ws/javaee/dd/web/common/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2010 IBM Corporation and others.
+ * Copyright (c) 2010,2022 IBM Corporation and others.
```

Comment:
Minor detail but usually there is a space after `,` in the date range.

Reply:
Fixed.